R/treeview.R

```diff
@@ -265,14 +265,7 @@ treeview <- function(e0,
   pldf <- pl[["data"]]
 
   suppressWarnings({
-    plot_cluster_sina(
-      pldf,
-      output_dir = output_dir,
-      mut_regexp = mutations,
-      lineage_regexp = lineages
-    )
-
-    for (vn in setdiff(branch_cols, c("logistic_growth_rate"))) {
+    for (vn in unique(c("logistic_growth_rate", branch_cols))) {
```

Comment:
where does the change `setdiff()` to `unique()` come from? These don't seem to give the same values for me:

```
branch_cols <- c("A", "B", "C")
unique(c("A", branch_cols)) # returns "A" "B" "C"
setdiff(branch_cols, "A") # returns "B" "C"
```

Reply:
This reduces the following:
```
f("a")

for (x in setdiff(some_vec, "a")) {
  f(x)
}
```

To this:

```
for (x in unique("a", some_vec)) {
  f(x)
}
```

Reply:
Thanks - that makes sense